apps/aws-amplify/src/components/common/AmplifyIcon.tsx

```diff
@@ -0,0 +1,29 @@
jcUSIhJ2mHGhJhu+BqHV6a5q6p22wrv3r8x75CW2wo0bUlHiueUl69NadC0JTc4ZOueN3xXQEvwkOjlext108XjkBCT9vbEFShTg4SYafAah0/KblRA2yqCXjJQF7kMmh/kc06gQrYwWo11mN8uBwJYIlrGS427537VwSrWtrybiZAtS4OE2MkObOASuU9LRsQJEGLpxUZmCAYK1LbYhadSIVsYreTvGhey9dnnxspCaBVwLxmongWmArUt6zgiZEtOY0KUGxayNf6LMIIFCNErLWzsXfVAUZC2lbRGEFegJA2aHxrnMH85zTAEtARnq/hmY8204bM9jJZ+AxGyNS6J0ZI3aJSX7EeogLPlvoF7yVj1WnItWut2mB+wNKhtdR1zXkOuM71TNqIC2lbp3cangBsK1LY4sSi453hFg9pWYxzm96ZFBwsRov/M0sZnAk3SMSE6hMN8naRBtDobEbJ18lelIMBolT5rRtpUoEBCTMibEyFbKkYr1YA1Du9/HawsRIhhHV4yXSefYinQtGUeTYRsKWlMiPKmdcvgmzAIoKOVK77enFuNrgK1raSzL3EFysSOVv0O82OfB4GAluBslV5r1n3ZnAK1LfWUPcVXoBYnEVr1O8y9pXyEChBi8MY9TdsQJWNC9C6lQrY0aH7oSp9Rn5dcDvrCImiVn+xoVq0fKBAthVgU3L2XgtFip82t5zrT534EChBi4dXmZfqPv17ChOjcQTrMkBA7eR0hWz3flnwBLVGTL9XjJQMOM0QrIW1ChGzp1bBoWunjB+4wf/e972O0isSyK6CgK1COAk1b0pE7itvetWmEVj0O8z3fhGvAomkZYW9HM2uExzFa63KYq2iRMcOZAYdsffJ54FfTIpu8V36s2SldrgJNW/I5Y8Qv1n6Z0KC2xQ8f4HWmD/L+qqrZ5PMvPdxkWC8frCK0aIf5MlODaHUmBrTGYeNPyxEoRIgf1O8lAw4zJER24QVih/kKWUPa1gAd5s++X3WwACGG9XrJQI1ZZiJHi3aYJ6pZTIgDcZiXflHKC2gJhOiWF3Y0v34wFIBWDYcZoTWQkK0vv88LYIloler3koHa8VYPE6JO7Nza5tgURkv+Vb8/+fkxzFfTos+WX7ivoxW1iSdjtLzNiJAtU9OQx56ubffo53fJb34esUKE+MGXHa2p26uFSNOiHeaLGUZL2bt/Uln4dSFP06rGFX7V3bINpjrWtuzfEyFbpgYJsVPpl8Pc/VU5AoUIMXxnxcdV9RNSP/zlT8T100zhj/bSQFGQh8QEP5G4AqVq0PzQP4f5xZ9XHSxEiTm/XF2ltau4VoV/qULlj/nqf16heKPQYZ7hKFDbsk6aK257chqiNUQ+CGc19J9KhQICi7bkkTuJdF6nK/4/2syAI7Roh/lsUIjaLw+DYf3z94UCfLQIWgQsMPiuMEF8BSqHCVE95WRx2xuXxGjJcMhWbzks9JtWDqHloLRsccb3846qIE2edJhPdzSIViYNrnEY/tHnEav2CrHwtPgKVKBAQqQc5pP34hrUttgWwzEv+fuwQNMCzhZ2y27dZ8uc87LYYTYVZNoiHeZDzLSGTFtdmMO89bflCFS7hegvEV879jkkxKQudpj3P0nHhJiE1jh89y/RwaKOVsuavB5MFF+BsjEhSkeKdXQiS2G0VCBk6/FvSiFNq4VCdMVfA9zoqpAQkw7hMP/eykJCzO5W22F+7fswRGg1v8kXbhFfgXI5RItymMffxTSIllzTYf7w82i+joUQbXGy4stzdGw09c4lQracbBYRYqaWw7zLp5+vfv6IQZMvTBaHbPkcoqUQDvMRv5Gz0PzAaqxx+PhfIlIxEaL53AXikC2LQ03eodY46GmMlrr9ur3kcjWs+mnVd7SI8eEW8GglrInEFSgJE2J67JgOuv61crAAITaalgiXOIpyx+Ueh6Yt6QoiZOvCJEZLHUmzevnHinsCN3kAVh1N3hWHnN7myZgQc8Si4JFOFhKitu2pdDRWNDZAtIK6mjw+bRV+IBxmDilR/u0wImRLwWjxvcnrTF+HYliFtgnRXiDsGo/fq2NC9AiHeXsrCwmxSxlBecnTwnpouXVM8vTREg9LU3zOkdmUyaTDjNFKEGscpla6e5yEaIlTYXsWOJgQHSpkS02vhAXQUg8ivOQKozg1ef92sY9kyBxq8vYhRMiWmYXaVlroMP9z5WDFS4iSeEHp3Nk5TIg64TCPPjZJC7GalvoLgWNbrhob4jRtGbcKx4czHJVDTd4jFgX/2spCbSuT2nztaKzK2BA7IRbmEQ4zh4TIx91EhWxhQlQO6Nv23vs5JKrB80P/HxK9BT3CuwAPSZgQbcJh3tJMQ0Ls4n3a3snffhBS1fZpK3ySCNnikBCZdDkRsqVjQkz22Y/+/r+EOKxWv0mMQuXFDjOH5gdrNhGypaQwIaprhGztGnnJRMXA2yLWFSw0ZOxsefOokK1sFpm2sqN2rfaSfy6QqGLgbZlGr3iNg8uhJq9e3EOEbLEsdLbks/t6yQCsdr3ACO4Wjg/3mTqHmjzlMB9kZbNIk+9im/zfK+jKE3R8hbhCHLIVcESItMN8g4wJke00vNpLpisOLzByT2wtdHaXmRxq8jbhMO8npTFaFYd5zLflsGa1W4jE5pAJPoeEmLAOFI8Px0mYEFNjx1R7yXTV/5BYvxCdh3YV3mxY7mFCNJeLHeY9rk2CR2vVGod7Kl5yrIWoFsTXgw7MyZgQcxPEQnwh6vHAbKptu010nYl4KGzJQyIuRNPtFV+BMjAhyncNI0K2GCZE/vcrveR/+x6rryv1+bTPV9e0D/5c0z4or1XIt6XFfvxIgg/mdwsXbpv5Pj+USohQn59F8Q6xDbieIfVdPV8pXillZXF7YseL//4f/4nVf6xd/7W6oj/pW/+9Vv3P6or+pG+9W7veelmctf/s02vWleJ6en3x0drx0itfWLM2FNeVm/4vT14upjkIA7kAAAAASUVORK5CYII="
```

Comment:
this is, just, ew. Will update this with actual asset(s) when applicable, possibly follow up PR for deploy checklist items

Reply:
Yes please!